features/294-login-without-specifying-login-id-key/api.md

```diff
@@ -1,3 +1,5 @@
+**This document is deprecated, refer to #323 instead**
```

Comment:
If you plan to mark this document at #294 is deprecated (which is a good idea), please make sure the new doc #323 is self-contained (i.e. people no need to read #294 first to understand #323 in future)

Reply:
This document should fully cover #294 (I'll add back the login ID APIs), except the vaguely specified designs:
- Pass codes
- SSO auto link